README.md

```diff
@@ -30,28 +30,31 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-# 📖 React-group-Project <a name="about-project"></a>
+# 📖 React-Capstone-Project <a name="about-project"></a>
 
-**React-group-Project** is a Project which is fetching images through base API  and displaying them on the app pages. User can reserve rockets and mission. User reservation will be displayed in the profile page. 
+**React-Capstone-Project** is a Project which displays a list of continents, their total population, and square area. It allows users to view in detail breakdown of this information by country of each continent. 
 
 ## 🛠 Built With <a name="built-with"></a>
 
-React Redux and API 
-
-### Tech Stack <a name="tech-stack"></a>
+React
+### React & Redux <a name="tech-stack"></a>
 
 ### Key Features <a name="key-features"></a>
 
-- **[React Redux and usage of API ]**
+- **[ Displays continent name, population and land area ]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"> </a>
 
-- [Live Demo](Will be available after final approval)
+- [Live Demo](https://reggeditt.github.io/react-capstone/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 💻 Video Presentation <a name="walk-through-video"></a>
+
+[Click](https://www.loom.com/share/2101666f3fb8443cb5f5b40d073f648b) to watch a video presentation on the project
+
 ## 💻 Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
```

Comment:
- [ ] Dear @Reggeditt, Great job on adding the video. But currently, I could not detect any sound while you are speaking. Please make sure your speech is out in the speaker. Please also don't forget to add all the detail points that are mentioned before.

Reply:
I'll re record is sir. thanks

Reply:
- [ ] As requested by the previous reviewers, kindly re record your video as the video you have submitted has  no sound. This is the [link](https://www.loom.com/share/2101666f3fb8443cb5f5b40d073f648b) you have provided.